Management/RuleSet.ruleset

```diff
@@ -5,22 +5,175 @@
   </Rules>
```

Comment:
I have no idea what is this.

Reply:
This was for customizing rulesets of StyleCop. I will look to see if there are better ways to customize it.

Reply:
Cool, so I will just let this be merged so you can take care of this later. This is not affceting the core logic.